src/storage/file_manager.c

```diff
@@ -1468,6 +1468,7 @@ file_header_dump_descriptor (THREAD_ENTRY * thread_p, const FILE_HEADER * fhead,
     case FILE_DROPPED_FILES:
     case FILE_VACUUM_DATA:
     default:
+      fprintf (fp, "\n");
```

Comment:
Hmm.. it seems that we need this also for the`FILE_BTREE` case when the `heap_get_indexinfo_of_btid()` returns an error.

Reply:
Oh, I see. I missed a part. I will make the correction.